plugins/Marketplace/PackageModel.py

```diff
@@ -250,3 +277,148 @@ def isCompatibleMaterialStation(self) -> bool:
     @pyqtProperty(bool, constant = True)
     def isCompatibleAirManager(self) -> bool:
         return self._is_compatible_air_manager
+
+    # --- manage buttons signals ---
+
+    stateManageButtonChanged = pyqtSignal()
+
+    installPackageTriggered = pyqtSignal(str)
+
+    uninstallPackageTriggered = pyqtSignal(str)
+
+    updatePackageTriggered = pyqtSignal(str)
+
+    enablePackageTriggered = pyqtSignal(str)
+
+    disablePackageTriggered = pyqtSignal(str)
+
+    recentlyInstalledChanged = pyqtSignal(bool)
+
+    # --- enabling ---
+
+    @pyqtProperty(str, notify = stateManageButtonChanged)
+    def stateManageEnableButton(self) -> str:
```

Comment:
This shouldn't be handled in the model, this is something that the view should handle

Reply:
Simplified it it is now basically a toggle between primary and secondary for all, I will rename these accordingly. Goes for all of these stateManage....